pkg/operator/ceph/reporting/status.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2021 The Rook Authors. All rights reserved.
```

Reply:
This stays, because I didn't create this file new. Seb created it, and I moved it in this pr.